YoCode/ProjectBuilder.cs

```diff
@@ -0,0 +1,90 @@
+﻿using System;
+using System.IO;
+using System.Text;
+
+namespace YoCode
+{
+    public class ProjectBuilder
```

Comment:
Most of the methods in this class are not used and there are no tests. Is there more code coming that calls these methods, or some tests coming? How do you know the methods work correctly if they're not used anywhere and there are no tests?

Reply:
These methods will be used for the Evidence object after we merge branch #40.
Also will be adding tests in the future. For now I tested the methods by running the program and outputting everything to console

Reply:
Okay, so you tested it with some temporary code that you didn't want to commit? And the tests are coming soon? I believe you... ;)